core/src/main/resources/IAF_Util/ConfigurationFileSystem.xml

```diff
@@ -0,0 +1,106 @@
+<module>	
+	<adapter name="FileSystem" description="Upload files to server" active="${fileSystem.active}">	
+		<receiver name="FileSystem">
+			<listener name="FileSystem"
+				className="nl.nn.adapterframework.http.rest.ApiListener"
+				uriPattern="fileSystem" 
+				method="POST"
+				multipartBodyName="destination"
+				authenticationMethod="AUTHROLE"
+				authenticationRoles="IbisDataAdmin"/>
+		</receiver>
+		<receiver name="FileSystem-jl">
+			<listener className="nl.nn.adapterframework.receivers.JavaListener" name="FileSystem-jl" serviceName="${fileSystem-jl.serviceName}" />
+		</receiver>
+
+		<pipeline firstPipe="switchCleanFilesystem">
```

Comment:
It is not necessary to set the firstPipe attribute, if it refers to the first pipe of the pipeline